frontend/app/stylus/collections.styl

```diff
@@ -141,3 +142,12 @@
 
   .svg-invalid
     color #D8653B
+
+  .upload-svg-select
+    cursor pointer
+    position absolute
+    top 0
+    left 0
+    width 100%
+    height 100%
+    opacity 0
```

Comment:
Because security issue which is not allowed simulating clicking on input element with ` type =file `  in some browsers, I set input element overlaying ` .svg-upload ` section as a workaround.